dnsapi/dns_ovh.sh

```diff
@@ -11,27 +11,14 @@
 
 #OVH_END_POINT=ovh-eu
 
-#'ovh-eu'
 OVH_EU='https://eu.api.ovh.com/1.0'
```

Comment:
don't remove the comments here.  just add the new endpoint.

Reply:
the envs are listed in _ovh_get_api so this is a duplication and noise